src/pages-helpers/curriculum/docx/builder/1_frontCover.ts

```diff
@@ -20,34 +22,44 @@ import { getSubjectIconAsset } from "@/image-data";
 
 export default async function generate(
   zip: JSZipCached,
-  { data }: { data: CombinedCurriculumData },
+  { data, slugs }: { data: CombinedCurriculumData; slugs: Slugs },
 ) {
   const iconKey = data.subjectTitle.toLowerCase();
 
   const sanityUrl = getSubjectIconAsset(iconKey)?.url;
   const images = await insertImages(zip, {
     icon: sanityUrl
       ? makeTransparentIfSanity(sanityUrl, cmToPxDpi(13))
-      : join(
+      : relative(
```

Comment:
Any reason for this change? I think this'll break in prod

Reply:
No, think it was a conflict from earlier! Will change it back